### ISSUE TITLE
Changing permissions for a team or person docs incorrect (or github UI has a bug)

### ISSUE THREAD

**Issue by Ben-Epstein** [github/docs#35511]
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-teams-and-people-with-access-to-your-repository#changing-permissions-for-a-team-or-person

### What changes are you suggesting?

The page suggests that within the collaborators section, you can click a dropdown to change the user's role. That dropdown doesn't exist (i believe it used to). The docs should be updated to reflect the UI change.

If this is a bug on the UI, I'm happy to report it to the current repo!

**Comment by Zelliekoos** [comment]
Great!!!

**Comment by nguyenalex836** [comment]
@Ben-Epstein Hello, and thanks for opening this issue! 👋 

> That dropdown doesn't exist (i believe it used to).

Would you mind sending over a screenshot of what you're seeing on your end once you get to this point? 💛

**Comment by Ben-Epstein** [comment]
Hey @nguyenalex836 Sure thing, here's what I see

---

![image](https://github.com/user-attachments/assets/4217184c-951e-4217-a34d-b475b702539d)

---
Thanks!

**Comment by nguyenalex836** [comment]
@Ben-Epstein Thank you! Always helpful to have a visual reference ✨ 

I checked a few different repos on my end, and it looks like the dropdown is still present for me - 

<img width="324" alt="image" src="https://github.com/user-attachments/assets/b350932e-481b-43e3-b48a-27a168000a61">

---------

<img width="415" alt="image" src="https://github.com/user-attachments/assets/eaa54121-e7e4-48f3-a428-39592ceb06af">

My guess is that this may be permissions issue, though we don't have the tools necessary to say for certain on this team. Our wonderful [support team](https://support.github.com/request) is best equipped to identify the reason you're not seeing the drop down ✨ If you reach out to them, they'll be happy to assist!

If after reaching out to them they identify a need to update our docs, please let us know! For now, I'll go ahead and close this issue 💛

**Comment by nguyenalex836** [comment]
@Ben-Epstein Good catch! Let me check with our SME team to see whether there are other nuances that would affect whether that dropdown appears - will return with more info 💛

**Comment by janbrasna** [comment]
FYI did check all repo+org types I could (visibility private/public, plan paid/free) and as _owner_ or _admin_ I do see the additional options (not just the role dropdown, but also team management buttons etc.) compared to OP's screen shot. @Ben-Epstein Any chance you just don't have the permissions necessary for changing others' access in the documented case?

**Comment by nguyenalex836** [comment]
@Ben-Epstein Apologies for the delay - given the proximity to the holidays, we've been unable to review as quickly as we typically do 💛 

Per @janbrasna's comment -

> Any chance you just don't have the permissions necessary for changing others' access in the documented case?

Have you had any luck checking whether this has been a factor on your end?

**Comment by Ben-Epstein** [comment]
Hey @nguyenalex836 yes I'm the owner of the repo, so I think my permissions must be 100%...

I converted the repo into an free Org and managed the permissions through the org itself, which did work, but that confuses me too

**Comment by ADNANALISALHADABY** [comment]
name: GitHub Actions Demo
run-name: ${{ github.actor }} is testing out GitHub Actions 🚀
on: [push]
jobs:
  Explore-GitHub-Actions:
    runs-on: ubuntu-latest
    steps:
      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
      - run: echo "🐧 This job is now running on a ${{ runner.os }} server hosted by GitHub!"
      - run: echo "🔎 The name of your branch is ${{ github.ref }} and your repository is ${{ github.repository }}."
      - name: Check out repository code
        uses: actions/checkout@v4
      - run: echo "💡 The ${{ github.repository }} repository has been cloned to the runner."
      - run: echo "🖥️ The workflow is now ready to test your code on the runner."
      - name: List files in the repository
        run: |
          ls ${{ github.workspace }}
      - run: echo "🍏 This job's status is ${{ job.status }}."

**Comment by nguyenalex836** [comment]
@Ben-Epstein Thanks for sharing your findings! Our SME team was able to review, and provided the following response:

> This is on a user owned repository right? We removed the dropdown there because you can only add users with write access. (For org owned forks the dropdown is still visible)
> 
> User-owned repo:
> <img width="918" alt="image" src="https://github.com/user-attachments/assets/e7e5e5b5-d69e-4e84-8d46-b87659123695" />
> 
> User owned fork of an org repo:
> <img width="972" alt="image" src="https://github.com/user-attachments/assets/0804b6da-1aac-40d5-a374-40619063e5a5" />
> 
> Free Org:
> <img width="1019" alt="image" src="https://github.com/user-attachments/assets/064d3214-1c71-42c6-a2fb-b117c4d8d70a" />

Does that provide some clarity regarding the dropdown? 💛

**Comment by nguyenalex836** [comment]
I'll go ahead and close this issue, but please let us know if you have any other questions 💛
